crypt-gpgme.c

```diff
@@ -3162,12 +3147,12 @@ print_dn_parts (FILE *fp, struct dn_array_s *dn)
 
 
 /* Parse an RDN; this is a helper to parse_dn(). */
-static const unsigned char *
-parse_dn_part (struct dn_array_s *array, const unsigned char *string)
+static const char *
+parse_dn_part (struct dn_array_s *array, const char *string)
 {
-  const unsigned char *s, *s1;
+  const char *s, *s1;
```

Comment:
@gahr Hm, for me its not obvious that they are not needed, can you explain it to me?^^

(btw. This wasn't meant as a review, i was just curious of your change -> I'm trying to understand mutt better :D)